examples/gno.land/p/demo/svg/svg.gno

```diff
@@ -0,0 +1,80 @@
+package svg
+
+import "gno.land/p/demo/ufmt"
+
+type Canvas struct {
+	Width  int
+	Height int
+	Elems  []Elem
+}
+
+type Elem interface{ String() string }
+
+func (c Canvas) String() string {
+	output := ""
+	output += ufmt.Sprintf(`<svg xmlns="http://www.w3.org/2000/svg" width="%d" height="%d">`, c.Width, c.Height)
+	for _, elem := range c.Elems {
+		output += elem.String()
+	}
+	output += "</svg>"
+	return output
+}
+
+func (c *Canvas) Append(elem Elem) {
+	c.Elems = append(c.Elems, elem)
+}
+
+type Circle struct {
+	CX, CY, R int
+	Fill      string
```

Comment:
maybe add a type Color to indicate that this should be a color, with some default consts? (ie. `ColorRed`, `ColorBlue`, or even just the color names)

Reply:
Fill is not limited to colors, it can contain things like `url(#gradientName)`, `none`, and probably other things I'm not aware.

I will add consts for colors anyway 👍